src/pages/index.tsx

```diff
@@ -1,16 +1,20 @@
-import { Button } from '@meemoo/react-components';
+import { Button, TextInput } from '@meemoo/react-components';
 import { GetServerSideProps, NextPage } from 'next';
 import { useTranslation } from 'next-i18next';
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
 import Head from 'next/head';
 import Link from 'next/link';
-import { useState } from 'react';
+import { KeyboardEvent, useState } from 'react';
+import { useQueryParams } from 'use-query-params';
 
-import { Hero, ReadingRoomCardList } from '@shared/components';
+import { HOME_QUERY_PARAM_CONFIG } from '@home/const';
+import { Hero, Icon, ReadingRoomCardList } from '@shared/components';
 import { sixItems } from '@shared/components/ReadingRoomCardList/__mocks__/reading-room-card-list';
 import { createPageTitle } from '@shared/utils';
 
 const Home: NextPage = () => {
+	const [searchValue, setSearchValue] = useState('');
+	const [, setQuery] = useQueryParams(HOME_QUERY_PARAM_CONFIG);
```

Comment:
```suggestion
	const [setQuery] = useQueryParams(HOME_QUERY_PARAM_CONFIG);
```

Reply:
Die eerste value gebruik ik niet vandaar de komma om direct het volgende item in de array te destructuren. 
Als ik deze aanpassing zou maken dan zou `setQuery` hier geen functie meer zijn maar het query object.